packages/ui/src/badge/stories/index.story.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryObj } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import '@wordpress/theme/design-tokens.css';
+
+/**
+ * Internal dependencies
+ */
+import { Badge } from '../index';
+
+const meta: Meta< typeof Badge > = {
+	title: 'Design System/Components/Badge',
+	component: Badge,
+	tags: [ 'status-experimental' ],
+};
+export default meta;
+
+type Story = StoryObj< typeof Badge >;
+
+export const Default: Story = {
+	args: {
+		children: 'Badge',
+	},
+};
+
+export const High: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'high',
+	},
+};
+
+export const Medium: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'medium',
+	},
+};
+
+export const Low: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'low',
+	},
+};
+
+export const Stable: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'stable',
+	},
+};
+
+export const Informational: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'informational',
+	},
+};
+
+export const Draft: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'draft',
+	},
+};
+
+export const None: Story = {
+	...Default,
+	args: {
+		...Default.args,
+		intent: 'none',
+	},
+};
+
+export const AllIntents: Story = {
+	...Default,
+	render: ( args ) => (
+		<div
+			style={ {
+				display: 'grid',
+				gridTemplateColumns: 'max-content min-content',
+				gap: '1rem',
+				color: 'var(--wpds-color-fg-content-neutral)',
```

Comment:
We'll probably want to assign a background color to ensure this shows well in "Dark" mode. Or not assign `color` at all. Or use `Box` .

Before|After
---|---
<img width="223" height="400" alt="image" src="https://github.com/user-attachments/assets/091102ae-c259-40e9-a09f-66964c408435" />|<img width="210" height="379" alt="image" src="https://github.com/user-attachments/assets/b500075e-ac60-452f-aed9-1285d473608f" />


```suggestion
				color: 'var(--wpds-color-fg-content-neutral)',
				backgroundColor: 'var(--wpds-color-bg-surface-neutral-strong)',
```